Source/WebKit/UIProcess/API/APIPageConfiguration.cpp

```diff
@@ -180,7 +180,7 @@ HashSet<WTF::String> PageConfiguration::maskedURLSchemes() const
 {
     if (m_data.maskedURLSchemesWasSet)
         return m_data.maskedURLSchemes;
-#if ENABLE(WK_WEB_EXTENSIONS)
+#if ENABLE(WK_WEB_EXTENSIONS) && PLATFORM(COCOA)
     if (webExtensionController() || weakWebExtensionController())
         return WebKit::WebExtensionMatchPattern::extensionSchemes();
```

Comment:
Looks like `WebExtensionMatchPattern` will be needed later.

Reply:
At some point yes, but I haven't written a c++ implementation of MatchPattern yet, so it can't be enabled on Linux yet. Almost all of the platform(cocoa) gates are missing implementations